grammars/javascript.cson

```diff
@@ -1023,6 +1004,29 @@
     ]
   }
   {
+    'begin': '\\?'
+    'beginCaptures':
+      '0':
+        'name': 'keyword.operator.ternary.js'
+    'end': ':'
+    'endCaptures':
+      '0':
+        'name': 'keyword.operator.ternary.js'
+    'patterns': [
+      {
+        'match': '(\\w+)(?=\\s*:)'
```

Comment:
See 157241d0fd3c8. There is no need to include `$self` here. Though most of the rules must be moved to the repo and it also requires addition testing, so I kept it as is for now. The second `$self` includes everything else between `?` and `:`